buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -618,7 +618,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def influxdb_version = "2.19"
     def httpclient_version = "4.5.13"
     def httpcore_version = "4.4.14"
-    def iceberg_bqms_catalog_version = "1.5.2-0.1.0"
+    def iceberg_bqms_catalog_version = "1.6.1-1.0.1-beta"
```

Comment:
Shall we add a TODO to move onto stable (non-beta) version before release cut?

Reply:
This is just the most recent BQMS version (the BigQuery team chose to name it like this). They'll let us know internally if/when they release a new version.